arm/Microsoft.DomainServices/AzureADDomainServices/.parameters/aadds.parameters.json

```
@@ -8,6 +8,14 @@
         "sku": {
             "value": "standard"
         },
+        "replicaSets": {
+            "value": [
+                {
+                    "location": "location-value",
```

I guess we'd have something like 'WestEurope' here?

```suggestion
                    "location": "WestEurope",
```